src/.github/workflows/ci.yml.jinja

```diff
@@ -185,7 +187,5 @@ jobs:
             {%- raw %}
             labels: ${{ steps.docker_meta_public.outputs.labels }}
             tags: ${{ steps.docker_meta_public.outputs.tags }}
-            build-args: |
-              VCS_REF=${{ github.sha }}
```

Comment:
I was thinking that we should let `docker_meta` handle everything tag-related. Would removing this cause any problems?

Reply:
After investigating a bit more, it seems this works on a different level and we could leave them both. 
Is it ok for you @Yajo?

Reply:
> we should let `docker_meta` handle everything tag-related

Yes, that's how it must be done.